platform/security/sts/security-sts-propertyclaimshandler/src/test/java/org/codice/ddf/security/sts/claims/property/TestAttributeFileClaimsHandler.java

```diff
@@ -44,120 +44,315 @@
+  }
+
+  @Test
+  public void testNoMatchRetrieveClaimValues() throws IOException {
+    // given
+    System.setProperty(SystemBaseUrl.HOST, "user1");
+
+    final AttributeFileClaimsHandler attributeFileClaimsHandler = new AttributeFileClaimsHandler();
+    attributeFileClaimsHandler.setAttributeFileLocation(getPathForValidTestAttributesFile());
+    attributeFileClaimsHandler.init();
+
+    final ClaimCollection claimCollection = getClaimCollectionForValidTestAttributesFile();
+
+    final ClaimsParameters unknownClaimsParameters = new ClaimsParameters();
+    final Principal principal = mock(Principal.class);
     when(principal.getName()).thenReturn("unknown");
     unknownClaimsParameters.setPrincipal(principal);
+
+    // when
+    final ProcessedClaimCollection processedClaims =
+        attributeFileClaimsHandler.retrieveClaimValues(claimCollection, unknownClaimsParameters);
+
+    // then
+    assertThat(processedClaims, is(empty()));
   }
 
   @Test
-  public void testGetSupportedClaimTypes() {
-    List<URI> supportedClaimTypes = attributeFileClaimsHandler.getSupportedClaimTypes();
-    assertTrue(supportedClaimTypes.contains(URI.create("test")));
-    assertTrue(supportedClaimTypes.contains(URI.create("test1")));
+  public void testRetrieveClaimsValuesNullPrincipal() throws IOException {
+    // when
+    System.setProperty(SystemBaseUrl.HOST, "user1");
+
+    final AttributeFileClaimsHandler attributeFileClaimsHandler = new AttributeFileClaimsHandler();
+    attributeFileClaimsHandler.setAttributeFileLocation(getPathForValidTestAttributesFile());
+    attributeFileClaimsHandler.init();
+
+    final ClaimsParameters claimsParameters = new ClaimsParameters();
+
+    final ClaimCollection claimCollection = new ClaimCollection();
+
+    // given
+    final ProcessedClaimCollection processedClaims =
+        attributeFileClaimsHandler.retrieveClaimValues(claimCollection, claimsParameters);
+
+    // then
+    assertThat(processedClaims, is(empty()));
   }
 
   @Test
-  public void testRetrieveClaimValuesAdmin() {
-    ProcessedClaimCollection processedClaims =
-        attributeFileClaimsHandler.retrieveClaimValues(claimCollection, adminClaimsParameters);
-    assertThat(processedClaims.size(), is(2));
-
-    for (ProcessedClaim processedClaim : processedClaims) {
-      if (processedClaim.getClaimType().toString().equals("test")) {
-        assertThat(processedClaim.getValues().size(), is(1));
-        assertThat(processedClaim.getValues().get(0), is("testValue"));
-      } else if (processedClaim.getClaimType().toString().equals("test1")) {
-        assertThat(processedClaim.getValues().size(), is(3));
-        assertTrue(processedClaim.getValues().contains("testing1"));
-        assertTrue(processedClaim.getValues().contains("testing2"));
-        assertTrue(processedClaim.getValues().contains("testing3"));
-      }
-    }
+  public void testGetSingleValuedSystemHighAttribute() throws IOException {
```

Comment:
❓For the tests related to the system high attributes, do you think we should use a separate file that contains a `system` user and security related attributes, e.g., `classification`, etc., instead of using generic user, attribute names and values? Might make it easier to understand what the tests are doing and testing.